aging_due_report/wizard/wizard.py

```diff
@@ -123,9 +124,8 @@ def _get_amount(self):
             for fn in field_names:
                 res[fn] = 0.0
             for doc in record.document_ids:
-                for fsum in field_sum:
-                    if fsum != 'not_due':
-                        res[fsum] += doc.read([fsum])[0][fsum]
+                for fsum in field_sum2:
```

Comment:
keeping same field_sum:
getting rid of conditional statement and
step back the indentation to as it is no longer a statement inside condition.